src/components/organisms/members.ts

```diff
@@ -14,6 +14,27 @@ export type Member = {
 }
 
 export const members: Member[] = [
+  {
+    isActive: true,
+    name: 'Arata',
+    imagePath: 'arata',
+    description: '',
```

Comment:
```suggestion
    description: '代表。情報科学類所属。主にインフラを担当。',
```
紹介文について何を言ったか忘れたので、こちらでお願いします